package.json

```diff
@@ -110,9 +110,9 @@
     "build-storybook": "storybook build -o docs-build",
     "test-storybook": "test-storybook",
     "storybook:axe": "yarn build-storybook && yarn storybook:axeOnly",
-    "storybook:axeOnly": "axe-storybook --build-dir docs-build",
+    "storybook:axeOnly": "axe-storybook --build-dir docs-build --timeout 10000",
```

Comment:
Thanks for adding this, @masoudmanson !

We can probably remove the following config in test files now we set the flag globally!

```ts
axe: {
  timeout: 10 * 1000,
},
```

Reply:
Ah, you're right. 
We don't even need a global timeout config. I'll revert this change and set a timeout for that specific component. 😯😥

#611